main.go

```diff
@@ -6,14 +6,15 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
-	"gobootcamp.com/controllers"
```

Comment:
For best practice separate the import libraries like:

Import(
// Standar library packages

// Our packages

// Third party libraries

)